src/components/main-form.js

```diff
@@ -1,180 +1,192 @@
 // References:
 // https://www.bankrate.com/calculators/mortgages/new-house-calculator.aspx
 // https://www.nerdwallet.com/mortgages/mortgage-rates/30-year-fixed
+// https://michaelbluejay.com/house/15vs30.html
 
 import _ from 'lodash';
 import React from 'react';
 import numeral from 'numeral';
-import { InputGroup, InputGroupAddon, InputGroupText, Input } from 'reactstrap';
+import styled, { css } from "react-emotion";
+import { Input } from 'reactstrap';
 import { FV, PMT } from 'formulajs/lib/financial';
 
+
+const COMPOUND_FREQUENCY = 12;
+
+const o = {
+  interestAmt: 0,
+  interestRate: 0,
+  term: 0,
+  pmt: 0,
+  fv: 0
+};
+
+let o1 = Object.assign({}, o);
+let o2 = Object.assign({}, o);
+
+o1.interestRate = 4.3;
+o1.term = 30;
+
+o2.interestRate = 4;
+o2.term = 15;
+
+const Row = styled('div')`
+  display: flex;
+  flex-direction: row;
+`
```

Comment:
bit confused by this syntax. Is there a string immediately after the `styled`?

Reply:
we can do a quick overview on emotion's syntax as part of our agenda this weekend. If you feel as uncomfortable with emotion.js as I am with functional CSS, we can always stick to our good'ol stylesheets.

Reply:
This looks like vanilla Javascript, but I'm unfamiliar w/ what

```javascript
something`
  foo bar
`
```

outputs